### ISSUE TITLE
Does the resnet32 have more than 32 convolutional layers?

### ISSUE THREAD

**Issue by hitcszx** [Chumsy0725/logit-adj-pytorch#4]
I have found that there are more than 32 convolutional layers in the implementations, is it right?

**Comment by hitcszx** [comment]
Moreover, the training parameters on CIFAR-10/-100  seem not consistent with those described in the paper, such as training epochs, learning rate decay, and weight decay.

**Comment by stablegradients** [comment]
I noticed it too. Is it resnet 56 Dr Menon? Since I noticed another paper of yours compares LA on res-56 against your CSL loss
